product-service/serverless.yml

```diff
@@ -11,10 +11,14 @@ provider:
   iamRoleStatements:
     - Effect: Allow
       Action:
-        - dynamodb:*
+        - "dynamodb:*"
+        - "sqs:*"
+        - "sns:*"
       Resource:
-        - arn:aws:dynamodb:ap-south-1:749453116506:table/products
```

Comment:
recommendation - It might be a good idea to define resources for a particular item (s3/sqs/sns)

Reply:
Right, it is a good practice to add particular methods. will update in future PRs